nextcloudappstore/api/v1/release/provider.py

```diff
@@ -35,6 +35,9 @@ def get_release_info(self, url: str, is_nightly: bool = False) -> Release:
             info = parse_app_metadata(meta.info_xml, self.config.info_schema,
                                       self.config.pre_info_xslt,
                                       self.config.info_xslt)
+            if meta.changes_xml:
+                whats_new = parse_app_whats_new(meta.changes_xml, self.config.changes_schema,
```

Comment:
Looks like an unused variable. If it's just for validation simply remove the assignment

Reply:
still WIP